packages/modeling/src/maths/plane/fromPoints.js

```diff
@@ -10,17 +10,39 @@ const vec3 = require('../vec3')
  * @returns {plane} out
  * @alias module:modeling/maths/plane.fromPoints
  */
-const fromPoints = (out, a, b, c) => {
-  const ba = vec3.subtract(vec3.create(), b, a)
-  const ca = vec3.subtract(vec3.create(), c, a)
-  vec3.cross(ba, ba, ca)
-  vec3.normalize(ba, ba) // normal part
-  const w = vec3.dot(ba, a)
+const fromPoints = (out, ...vertices) => {
```

Comment:
Use the spread operation is sexy but it seems kind of a performance hit.

`Newplane = plane.fromPoints(…polygon.vertices)`

Reply:
I didn't want to break a widely used API. Maybe a new `plane.fromPointArray` function? Or do a type check if the parameter is an array?